rotkehlchen/data_import/importers/bitstamp.py

```diff
@@ -0,0 +1,112 @@
+import csv
+from pathlib import Path
+from typing import Any
+
+from rotkehlchen.assets.converters import asset_from_bitstamp
+from rotkehlchen.constants.assets import A_USD
+from rotkehlchen.data_import.utils import BaseExchangeImporter
+from rotkehlchen.db.drivers.gevent import DBCursor
+from rotkehlchen.errors.asset import UnknownAsset
+from rotkehlchen.errors.misc import InputError
+from rotkehlchen.errors.serialization import DeserializationError
+from rotkehlchen.exchanges.data_structures import AssetMovement, AssetMovementCategory, Trade
+from rotkehlchen.serialization.deserialize import (
+    deserialize_asset_amount,
+    deserialize_fee,
+    deserialize_timestamp_from_date,
+)
+from rotkehlchen.types import Location, Price, TradeType
+
+
+class BitstampTransactionsImporter(BaseExchangeImporter):
+    def _consume_bitstamp_trade(
+            self,
+            write_cursor: DBCursor,
+            csv_row: dict[str, Any],
+            timestamp_format: str = '%b. %d, %Y, %I:%M %p',
+    ) -> None:
+        """
+        Consume the file containing only trades from Bitstamp.
+        - UnknownAsset
+        - DeserializationError
+        """
+        timestamp = deserialize_timestamp_from_date(
+            date=csv_row['Datetime'],
+            formatstr=timestamp_format,
+            location='Bitstamp',
+        )
+
+        amount, amount_symbol = csv_row['Amount'].split(' ')
```

Comment:
this can raise `KeyError`. You already handle it but state it in the comment

Reply:
Also this can raise `ValueError` if there is no enough values to unpack. That should be also be handled/stated in the docstring